storage/src/vespa/storage/bucketdb/striped_btree_lockable_map.h

```diff
@@ -0,0 +1,88 @@
+// Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+#pragma once
+
+#include "btree_lockable_map.h"
+#include <memory>
+#include <vector>
+
+namespace storage::bucketdb {
+
+// Bucket database implementation that stripes all superbuckets across
```

Comment:
Nitpick: Consider `/** */` for class comments as used elsewhere.

Reply:
Done